CHANGELOG.unreleased.md

```diff
@@ -34,6 +35,7 @@
 <!--packages-start-->
 
 - @xen-orchestra/xapi minor
+- @xen-orchestra/backups minor
```

Comment:
Try keeping this list ordered in the future because it causes issues when rebasing other branches :slightly_smiling_face: